sonar-db/src/main/java/org/sonar/db/component/ResourceMapper.java

```diff
@@ -19,15 +19,9 @@
  */
 package org.sonar.db.component;
 
-import java.util.List;
 import org.apache.ibatis.annotations.Param;
 
 public interface ResourceMapper {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Annotate the "ResourceMapper" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1609)